go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift-knative/serverless-operator
 
-go 1.21
+go 1.22.0
+
+toolchain go1.22.4
```

Comment:
not sure if this still will cause issues or not. But it will be added eventually by dependabot.

cc @skonto @pierDipi @creydr @matzew